torba/baseaccount.py

```diff
@@ -205,7 +205,9 @@ def __init__(self, ledger: 'baseledger.BaseLedger', wallet: 'basewallet.Wallet',
         self.id = public_key.address
         self.name = name
         self.seed = seed
+        self.password = None
         self.encrypted = encrypted
+        self.encrypted_on_disk = encrypted
```

Comment:
`disk` is such an archaic word :-P also, the only place where this is referenced is in `to_dict()` which doesn't necessarily mean that something is being written to disk, maybe it would be more accurate to call this something like `serialize_encrypted` (which goes better with it's use in `to_dict()`). what do you think?

Reply:
:+1: